apps/mobile-mzima-client/src/app/map/components/post-item/post-item.component.html

```diff
@@ -20,7 +20,7 @@
       fill="clear"
       shape="normal"
       color="medium"
-      *ngIf="!checkbox"
+      *ngIf="isLoggedIn && !checkbox"
       class="post__details"
```

Comment:
@MMwandigha You could solve the flicker of the dots with creating a function checking the status directly instead of assigning a variable in `onInit`. Something like this:
```
public getLoggedInStatus() {
    return this.sessionService.isLogged();
  }
```

In that way, it would check immediately instead of waiting for the variable to be assigned/reassigned, I think thats what causes the flicker.

Reply:
@Angamanga I see, let me try doing this instead